WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -125,7 +125,6 @@ decisions = {
 					NOT = { culture = ROOT }
 				}
 			}
-			holy_order = no
```

Comment:
Rulers of holy orders shouldn't convert to local cultures, and this is a vanilla file, so I don't see the need for modifying that.